### ISSUE TITLE
Speed of the stream

### ISSUE THREAD

**Issue by SmartHank1** [mcfisto/homebridge-homepod-radio#9]
Hey McFisto,
Just closed my last issue, found the problem...
Again thx for this great plugin.
I have new question...
The stream I play sounded quite slow to me, the voices seemed a little bit weird.
I let the stream play for 3 hours and 45 minutes now and it played the news of 11:30.
So there is a delay of roughly 7%...
Any chance to fix this?

All the best

**Comment by DJay-X** [comment]
Same issue here on HomePod. Voices sound not natural and more deeper, because of the stream is slower than original. I did a comparison with a self created mp3 with voice announcement and the voice playing through the plugin to HomePod is much deeper than the original source, when playing the mp3 on the computer.

Thanks in advance @mcfisto for considering a next update and the time you are investing.

**Comment by toppe** [comment]
Had a similar problem: the voices are like Donald Duck, speed was to high.
To correct this I've used [Audacity](https://www.audacityteam.org/download/), created a new file (stereo!) and put the bitrate to 128. Then dag & drop your file into the new created file. Then export as MP3 and be sure that the type is stereo!

**Comment by mcfisto** [comment]
Yep, I have the same problem. Will fix this ASAP. It seems to me like a problem with lame decoder.

**Comment by DJay-X** [comment]
@mcfisto 
Today I did some Voice announcement as a mp3 with original Siri German male voice.
Sounds completely different streaming with homebridge-airtunes-radio
Example can be find here. https://youtu.be/d5jXNs4HEos
Hope you can help that the output is going to be the same as the input mp3 file. 
Thanks again for your great work and the time you are investing.

https://www.dropbox.com/s/5ds7byi127mu3ac/Backofen.mp3?dl=0

**Comment by neilheyes** [comment]
I have the speed issue with a stream and the other one isn't playing.

http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/http-icy-mp3-a/vpid/bbc_6music/format/pls.pls - doesn't play - although is ok in my browser (just to check!).

http://icecast1.rte.ie/ieradio1 - just comes out too fast!

Where would the debug information be?
